CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * Fixed an issue where the map’s floating buttons are misaligned with its attribution button and sometimes untappable after rotating the device during turn-by-turn navigation. ([#3776](https://github.com/mapbox/mapbox-navigation-ios/pull/3776))
 * Fixed an issue where the top instruction banner couldn't be swiped back to the current one. ([#3785](https://github.com/mapbox/mapbox-navigation-ios/pull/3785))
 * Fixed an issue where the road name label shows wrong shield image when applying custom style under poor network connection. ([#3792](https://github.com/mapbox/mapbox-navigation-ios/pull/3792))
+* Added a `SpeedLimitView.shouldShowUknownSpeedLimit` property which defines the view behavior if the `SpeedLimitView.speedLimit` property is `nil`. Setting this flag to `true` will cause the view to display `"--"` as a speed limit instead of being invisible. ([#3798](https://github.com/mapbox/mapbox-navigation-ios/pull/3798))
```

Comment:
`shouldShowUnknownSpeedLimit` has a typo throughout.

```suggestion
* Added a `SpeedLimitView.shouldShowUnknownSpeedLimit` property which defines the view behavior if the `SpeedLimitView.speedLimit` property is `nil`. Setting this flag to `true` will cause the view to display `"--"` as a speed limit instead of being invisible. ([#3798](https://github.com/mapbox/mapbox-navigation-ios/pull/3798))
```